### ISSUE TITLE
Accessibility: Can't focus Accessible View with JAWS 

### ISSUE THREAD

**Issue by rohanshardha** [microsoft/vscode#194446]
Type: <b>Bug</b>

Wehn pressing alt+F2 JAWS doesn't move focus to accessible view in notebooks. The jaws focus remains on the cell whyich was last edited.. Tried with JAWS 2023.2307 in VS Code 1.82.2 and VS Code insiders 1.83.0. It did work until version 1.81.1

VS Code version: Code - Insiders 1.83.0-insider (623feb6ddd360d97dc77e2ff577dd2c549ef442d, 2023-09-28T11:51:23.482Z)
OS version: Windows_NT x64 10.0.22621
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|11th Gen Intel(R) Core(TM) i7-1185G7 @ 3.00GHz (8 x 2995)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|15.84GB (7.58GB free)|
|Process Argv|--crash-reporter-id 15dd5adc-8603-4b49-b9c9-01a6da5e3e62|
|Screen Reader|yes|
|VM|0%|
</details><details><summary>Extensions (7)</summary>

Extension|Author (truncated)|Version
---|---|---
python|ms-|2023.16.0
vscode-pylance|ms-|2023.9.30
jupyter|ms-|2023.8.1002501831
jupyter-keymap|ms-|1.1.2
jupyter-renderers|ms-|1.0.17
vscode-jupyter-cell-tags|ms-|0.1.8
vscode-jupyter-slideshow|ms-|0.1.5

</details><details>
<summary>A/B Experiments</summary>

```
vsliv695:30137379
vsins829:30139715
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythontb:30258533
pythonptprofiler:30281269
vshan820:30294714
vscod805:30301674
bridge0708:30335490
bridge0723:30353136
vsaa593:30376534
pythonvs932:30404738
py29gd2263:30784851
vsclangdf:30492506
c4g48928:30535728
dsvsc012:30540252
pynewext54:30618038
showlangstatbar:30737417
57b77579:30687741
pythonfmttext:30716741
fixshowwlkth:30771523
showindicator:30805243
pythongtdpath:30726887
i26e3531:30792625
welcomedialog:30812478
pythonnosmt12:30779711
pythonidxpt:30768918
pythonnoceb:30776497
copilotsettingt:30808721
asynctok:30821568
dsvsc013:30777762
dsvsc014:30777825
diffeditorv2:30786206
pythonlinttype:30823781
pythonmpsinfo:30842935
dsvsc015:30821418
pythontestfixt:30826906
pythonfb280951:30830809
pythonregdiag:30842812
pythoncet0:30848247

```

</details>

**Comment by meganrogge** [comment]
does `alt+f2` work when focus is in the terminal?

**Comment by vadimkholodilo** [comment]
The same happens for Orca in Linux.

**Comment by meganrogge** [comment]
If anyone on linux could suggest a keybinding here that would work, I'd appreciate it. I don't have linux, so cannot test this.

**Comment by meganrogge** [comment]
@rohanshardha does `alt+f2` work when focus is in the terminal? does it work if you use NVDA?

**Comment by vadimkholodilo** [comment]
I don’t know about the terminal. However, when Github Copilot offers me suggestions, I can press Alt+F2 on Windows and it works. However, On Linux, when I press Shift+F2, it doesn’t. (On Linux the key combination is different).

You can also try to open the accessible view from the command palett (CTRL+p). When I do it on Linux, nothing happens.

I use Ubuntu 23.04, Orca 44, and the latest stable version of VSCode (Visual Studio Code September 2023 <https://code.visualstudio.com/updates/v1_83> )

**Comment by vadimkholodilo** [comment]
Shift + F2

**Comment by meganrogge** [comment]
/duplicate #192090

`Ctrl+up` in the terminal should work. If you have a suggestion for a keybinding that works for linux, I'd be very open to it.

**Comment by rohanshardha** [comment]
When pressing Alt+F2 opens accessible view only after pressing control+down arrow in a notebook in edit mode. Pressing Alt+F2 in the terminal opens accessible view in VS Code. JAWS doesn't read the output of a cell after pressing control+down arrow in a notebook cell but NVDA does. You can access the output of the cell with JAWS by pressing control+down arrow in a notebook cell and then pressing Alt+F2 to open the accessible view and read the output. 

@meganrogge

**Comment by vadimkholodilo** [comment]
Yeah, that’s in the terminal. I’m talking about the case, when you are in the code editor and try to open Github Copilot suggestions by pressing Shift+F2. I probably should create a separate issue for that

**Comment by meganrogge** [comment]
@rohanshardha Sorry, I am still not understanding the issue.
